doc/contributing.md

```diff
@@ -16,37 +16,217 @@ and we encourage all to read it carefully.
+
+After you've submitted a pull request, you should expect to hear at least a
+comment within a couple of days. We may suggest some changes, improvements or
+alternative implementation details.
+
+To increase the chances of getting your pull request accepted quickly, try to:
+
+* Submit a friendly PR
+  - Write a good and detailed description of what the PR does.
+  - Write some documentation for your code (docstrings) and leave comments
+    explaining the *reason* behind non-obvious things.
+  - Write tests for the code you wrote/modified if needed.
+    Please refer to [Testing your code](#testing-your-code) or
+    [Testing plots](#testing-plots).
+  - Include an example of new features in the gallery or tutorials.
+    Please refer to [Gallery plots](#gallery-plots) or [Tutorials](#tutorials).
+* Have a good coding style
+  - Use readable code, as it is better than clever code (even with comments).
+  - Follow the [PEP8](http://pep8.org) style guide for code and the
+    [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html)
+    for docstrings. Please refer to [Code style](#code-style).
+
+Pull requests will automatically have tests run by GitHub Actions.
+This includes running both the unit tests as well as code linters.
+GitHub will show the status of these checks on the pull request.
+Try to get them all passing (green).
+If you have any trouble, leave a comment in the PR or
+[get in touch](#how-can-i-talk-to-you).
+
+## Contributing Documentation
+
+### PyGMT Documentation Overview
+
+There are four main components to PyGMT's documentation:
+
+* Gallery examples, with source code in Python `*.py` files under the
+  `examples/gallery/` folder.
+* Tutorial examples, with source code in Python `*.py` files under the
+  `examples/tutorials/` folder.
+* API documentation, with source code in the docstrings in Python `*.py`
+  files under the `pygmt/src/` and `/pygmt/datasets/` folders.
+* Getting started/developer documentation, with source text in ReST `*.rst`
+  and markdown `*.md` files under the `doc/` folder.
+
+The documentation are written primarily in
+[reStructuredText](https://docutils.sourceforge.io/rst.html) and built by
+[Sphinx](http://www.sphinx-doc.org/). Please refer to
+[reStructuredText Cheatsheet](https://docs.generic-mapping-tools.org/latest/rst-cheatsheet.html)
+if you are new to reStructuredText. When contributing documentation, be sure to
+follow the general guidelines in the [pull request workflow](#pull-request-workflow)
+section.
+
+There are two primary ways to edit the PyGMT documentation. For simple documentation
+changes, you can easily
+[edit the documentation on GitHub](#editing-the-documentation-on-github). This
+only requires you to have a GitHub account. For more complicated changes, you
+can [edit the documentation locally](#editing-the-documentation-locally). In
+order to build the documentation locally, you first need to
+[set up your environment](#setting-up-your-environment).
```

Comment:
How about using a list like **There are four main components to PyGMT’s documentation:** above?
```suggestion
There are two primary ways to edit the PyGMT documentation:
- For simple documentation changes, you can easily
  [edit the documentation on GitHub](#editing-the-documentation-on-github).
  This only requires you to have a GitHub account.
- For more complicated changes, you can
  [edit the documentation locally](#editing-the-documentation-locally).
  In order to build the documentation locally, you first need to
  [set up your environment](#setting-up-your-environment).
```